source/common/json/config_schemas.cc

```diff
@@ -899,6 +899,7 @@ const std::string Json::Schema::TOP_LEVEL_CONFIG_SCHEMA(R"EOF(
         "type" : "object",
         "properties" : {
           "access_log_path" : {"type" : "string"},
+          "profiler_path" : {"type" : "string"},
```

Comment:
You can also defer the `config_schemas.cc` change to your next PR, since it's not used yet here.